packages/xo-server-sdn-controller/src/index.js

```diff
@@ -349,10 +394,10 @@ class SDNController extends EventEmitter {
       MTU: 0,
       other_config: {
         automatic: 'false',
```

Comment:
What is this entry?

Reply:
Inherited from here:
https://github.com/vatesfr/xen-orchestra/blob/e8cd1e070fc5674089ca3404fc62ea775b7c3586/packages/xo-server/src/xapi/index.js#L2043

It's to avoid confusion by XenCenter.